ssh/tailssh/incubator_darwin.go

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) 2022 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//go:build darwin
+// +build darwin
```

Comment:
FYI these lines aren't necessary when the filename already ends in a common thing like `_darwin.go`.